### ISSUE TITLE
Docstrings

### ISSUE THREAD

**Issue by ax3l** [skuschel/postpic#51]
Argh, I really wish methods in classes like `plotting.plottercls` would have doc-strings ;)

**Comment by skuschel** [comment]
Yes absolutely right! However -- the plotter class seems lowest priority since Its internal design is pretty bad. I would like to have something which can work with jupyter notebooks as well. For everything else I agree on the imortance, but havent had much time for that yet.

**Comment by ax3l** [comment]
Yes, the issue actually more about using Docstring in general for any public API :)
